src/resources/js/v2/tickets-commerce.js

```diff
@@ -84,6 +87,41 @@ tribe.tickets.commerce = {};
 		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
 	}
 
+	/**
+	 * Display the notice component.
+	 *
+	 * @since TBD
+	 *
+	 * @param {Object} data Data for notice.
+	 */
+	obj.showNotice = function( data = {} ) {
+		const $notice = $( obj.selectors.noticeItem );
+		const $content = $notice.find( obj.selectors.noticeContent );
+		const $title = $notice.find( obj.selectors.noticeTitle );
+
+		if ( ! $notice.length || ! $content.length || ! $title.length ) {
+			return;
+		}
+
+		const title = data.title ? data.title : tecTicketsCommerceData.default_notice.title;
+		const message = data.message ? data.message : tecTicketsCommerceData.default_notice.content;
+
+		$title.text( title );
+		$content.text( message );
+		$notice.show();
+	}
```

Comment:
This is OK for now, but as this JS becomes more complex, we're going to want to split these into smaller functions. To me, `showNotice()` means that the notice will be displayed. It doesn't mean to me that the title or content will be set. This should be set in a separate function so that we can show and hide the notice without having to set the title and content each time.